src/from-scratch.js

```diff
@@ -1,13 +1,86 @@
 /* eslint-disable max-classes-per-file */
 class Circle {
-
-}
-
-class BankAccount {
-
-}
-
-module.exports = {
+  constructor(radius, color){
+    this.radius = radius;
+    this.color = color;
+  }
+  /*
+  There's different ways to get the same solution.
+  To get the area of a circle, we multiply PI by the radius squared.
+  Math.PI gives us the value of PI, and we square the radius using **, which acts as an exponent.
+  Another option is to use Math.pow(), which takes two arguments: a number and the exponent, and returns the result.
+  You could also square the radius by multiplying it by itself (this.radius * this.radius), but that only works for squaring.
+  */
+  getArea() {
+    return Math.PI * this.radius ** 2;
+  }
+  getCircumference() {
+    return 2 * Math.PI * this.radius;
+  }
+  draw(){
+    return `Drawing a ${this.color} circle.`
+  }
+  /*
+  The `this` keyword refers to the current instance of the class.
```

Comment:
This comment can be moved above to help explain the draw method.
```suggestion
  The `this` keyword refers to the current instance of the class. Below, we reference the `color` property of the current instance that is invoking this method.
  draw(){
    return `Drawing a ${this.color} circle.`
  }
  /*
```